nurseconnect/urls.py

```diff
@@ -4,6 +4,9 @@
 from django.conf.urls import patterns, include, url
 from django.conf.urls.static import static
 from django.contrib import admin
+from django.contrib.auth.decorators import login_required
+from nurseconnect.views import NurseConnectEditProfileView, \
```

Comment:
Consider breaking up these imports into two to avoid the line break, or use a tuple like so:
```
from nurseconnect.views import (NurseConnectEditProfileView,
    NurseConnectRegistrationView)
````